src/pixie/fileformats/qoi.nim

```diff
@@ -1,5 +1,4 @@
-import std/endians, chroma, flatty/binny
```

Comment:
morepretty does not understand [] imports so we do not use them (support could be added tho, not opposed to them if it is automatic)